server/routes/RedirectRoutes.ts

```diff
@@ -60,4 +60,11 @@ export const RedirectRoutes = (config: ServerConfig, clientConfig: ClientConfig)
   router.get('/version/?', (_req, res) => {
     return res.json({version: config.VERSION});
   }),
+  router.get('/oidc/?', (_req, res) => {
+    const {query} = _req;
+    const queryString = Object.keys(query)
+      .map(key => `${encodeURIComponent(key)}=${encodeURIComponent(query[key] as string)}`)
+      .join('&');
+    return res.redirect(HTTP_STATUS.MOVED_PERMANENTLY, `/auth/#/oidc${queryString ? '?' : ''}${queryString}`);
```

Comment:
im confused why we add an new oidc endpoint and include a moved_permanently status

Reply:
This is because, if we use the redirect mode for oauth (we will see if we use this in the future, rohan wants me to test popup mode), has router routes will not be accepted by every oauth provider (e.g. google).

The route xyz.com/#/abc will be declined because it is against the schema, so we need to use xyz.com/abc.
Our app is not running under xzy.com/abc, so we have to redirect with our node server to the right route.